README_v_0_2_x.md

```diff
@@ -38,21 +43,29 @@
 
 ## Installation
 
-The easiest way to install `pyani` is to use `pip3`:
+The easiest way to install `pyani` v0.2 is to use `conda` or  `pip`. `conda` is recommended for the simplest installation of third-party tool dependencies (`mummer` and `BLAST`/`BLAST+`).
+
+### `conda`
+
+You will need to install the `bioconda` channel, following instructions at [https://bioconda.github.io/user/install.html](https://bioconda.github.io/user/install.html). Then, to create a new environment for `pyani` and install the program, issue the following command:
 
 ```bash
-pip3 install pyani
+conda create --name pyani_env python=3.8 -y
+conda activate pyani_env
+conda install pyani
 ```
 
-From version 0.1.3.2 onwards, this should also install all the required Python package dependencies. Prior to this version (i.e. 0.1.3.1 and earlier), you can acquire these dependencies with `pip -r`, and pointing at `requirements.txt` from this repository:
+### `pip`
+
+`pip` will install `pyani` and its Python dependencies, but not the third-party tools.
 
 ```bash
-pip3 install -r requirements.txt
+pip3 install pyani
 ```
 
 ## Docker images
 
-`pyani`'s scripts are also provided as [Docker](https://www.docker.com/) images, that can be run locally as containers. To use these images, first install Docker, then to run the corresponding scripts issue either:
+`pyani` v0.2 scripts are also provided as [Docker](https://www.docker.com/) images, that can be run locally as containers. To use these images, first install Docker. Then, to run the corresponding scripts, issue either:
 
 ```bash
 docker run -v ${PWD}:/host_dir leightonpritchard/average_nucleotide_identity
```

Comment:
@widdowquinn
I've never used Docker. Should this really have a directory named for you in the example?

Reply:
Yes. That's the name of the account that hosts the image on DockerHub (see, e.g. [here](https://hub.docker.com/search?q=leightonpritchard&type=image)). The command pulls and runs `leightonpritchard/average_nucleotide_identity` from Dockerhub.

I'd be open to having a neutral DockerHub account to host these tools. Possibly this would be best under the collective genomeRxiv name.